src/components/OSCALLoader.js

```diff
@@ -100,6 +97,11 @@ export default function OSCALLoader(props) {
   );
 }
 
+function urlToLoad(defaultUrl) {
```

Comment:
Could this be named better? Maybe documentation on purpose would help?

Reply:
Looking at the function, it makes sense to rename it, will do that, comment on its purpose, and push a new commit.